workflows_api/runtime/src/config.py

```diff
@@ -1,19 +1,15 @@
-from pydantic import AnyHttpUrl, BaseSettings, Field, constr
+from pydantic import BaseSettings, Field, constr
 
 AwsArn = constr(regex=r"^arn:aws:iam::\d{12}:role/.+")
 AwsStepArn = constr(regex=r"^arn:aws:states:.+:\d{12}:stateMachine:.+")
 
 
 class Settings(BaseSettings):
-    jwks_url: AnyHttpUrl = Field(
-        description="URL of JWKS, e.g. https://cognito-idp.{region}.amazonaws.com/{userpool_id}/.well-known/jwks.json"  # noqa
-    )
     data_access_role_arn: AwsArn = Field(  # type: ignore
         description="ARN of AWS Role used to validate access to S3 data"
     )
-    userpool_id: str = Field(description="The Cognito Userpool used for authentication")
-    client_id: str = Field(description="The Cognito APP client ID")
-    client_secret: str = Field("", description="The Cognito APP client secret")
```

Comment:
don't need client_id/client_secret since we're differing the token generation to `veda-backend`

`userpool_id` and `jwks_url` can be derived from `workflows_client_secret_id`